aries-site/src/data/structures/components.js

```diff
@@ -575,6 +575,7 @@ export const components = [
   },
   {
     name: 'Select',
+    accessibility: 'Passed WCAG 2.2 AA',
```

Comment:
Since we are not passing rule 2.4.7 I think we should change this to
```suggestion
    accessibility: 'Failed WCAG 2.2 AA',
```